pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -182,7 +185,14 @@ func (hfh *HttpForwarderHandlerV2) EstimatedTags() int {
 
 // DispatchMetricMap dispatches a metric map to the MetricConsolidator
 func (hfh *HttpForwarderHandlerV2) DispatchMetricMap(ctx context.Context, mm *gostatsd.MetricMap) {
-	hfh.consolidator.ReceiveMetricMap(mm)
+	select {
+	case <-ctx.Done():
```

Comment:
There is a point that both context and done are ready so it can randomly return either, does that matter here?

Reply:
Nah, the only variability is in the logs, there's no meaningful observable difference - also they're both debug lines.